### PR TITLE
changes for deleting the city

### PR DESCRIPTION
**Pull request by sridhar07**
Note: changes for updating the boardingPoints instead of changes for deleting the city.

### CODE REVIEW

File: src/main/webapp/js/controllers/boardingPointsListController.js [hunk 1]
```diff
@@ -42,6 +50,20 @@ angular.module('myBus.boardingPointModule', [])
...
         };
+        $scope.handleClickAddBoardingPoint = function () {
+            var modalInstance = $modal.open({
+                templateUrl: 'add-boardingpoint-to-city-state-modal.html',
+                controller: 'AddBoardingPointController',
+                resolve: {
+                    cityId:function(){
+                        return $scope.cityId;
+                    },
+                    city: function() {
```
Comment (srinikandula) [inline src/main/webapp/js/controllers/boardingPointsListController.js]: Why are you passing city here? Don't only need cityId in AddBoardingPointController?

File: src/main/webapp/js/controllers/boardingPointsListController.js [hunk 2]
```diff
@@ -127,33 +130,31 @@ angular.module('myBus.boardingPointModule', [])
     })
 
     //======================Model - DeleteBoardingPointController=============================================
-.controller('DeleteBoardingPointController', function ($scope, $modalInstance, $http, $log,$route, deleteCityId, cityManager) {
+    .controller('DeleteBoardingPointController', function ($scope, $modalInstance, $http, $log,$route, deleteCityId, cityManager) {
         $scope.id = deleteCityId;
         $scope.ok = function (id) {
         cityManager.deleteCity(id,function(data) {
```
Comment (srinikandula) [inline src/main/webapp/js/controllers/boardingPointsListController.js]: Are you deleting the city or the boardingpoint? You should be deleting the boardingpoint not city.

**Comment by srinikandula** [comment]
Sreedhar I see there is unused code blocks in this. Can you please cleanup these? I'll wait for you to fix these.

**Comment by sridhar07** [comment]
Ok Srini,I will fix them.

**Pull request closed without merging**